garak/cli.py

```diff
@@ -337,7 +352,7 @@ def main(arguments=None) -> None:
     import garak.evaluators
 
     try:
-        plugin_types = ["probe", "generator"]
+        plugin_types = ["probe", "generator", "harness"]
```

Comment:
@jmartin-tech Does this imply harnesses need to implement Configurable?

Reply:
They already do see: https://github.com/leondz/garak/blob/4c4a3826020fe7423eb5033cd8600d506219ab15/garak/harnesses/base.py#L27-L36